static/usr/share/vdsm/autounattend/Autounattend.xml.in

```diff
@@ -1,3 +1,8 @@
+<!--                                                                                                                                                                                                                                                                                                                         
+SPDX-FileCopyrightText: Red Hat, Inc.                                                                                                                                                                                                                                                                                        
+SPDX-License-Identifier: GPL-2.0-or-later                                                                                                                                                                                                                                                                                    
+-->  
```

Comment:
xml comment looks good, I hope other xmls files can use the same solution instead of adding extra .license file.

Reply:
I was also a bit disappointed when I saw that it made tests fail :/